src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+
+        try (BufferedReader reader = Files.newBufferedReader(Paths.get(fromFileName))) {
+            String line;
+
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(",");
+                switch (parts[0]) {
+                    case "supply":
+                        supply += Integer.parseInt(parts[1]);
+                        break;
+                    case "buy":
+                        buy += Integer.parseInt(parts[1]);
+                        break;
+                    default:
+                        throw new RuntimeException("Something went terribly "
+                                + "wrong in switch-case block");
+                }
+            }
+
+            int result = supply - buy;
+
+            StringBuilder report = new StringBuilder();
+            report.append("supply,").append(supply).append(System.lineSeparator());
+            report.append("buy,").append(buy).append(System.lineSeparator());
+            report.append("result,").append(result).append(System.lineSeparator());
 
+            try (BufferedWriter writer = Files.newBufferedWriter(Paths.get(toFileName))) {
```

Comment:
please create private helper methods for reading and writing and move logic of reading and writing to these methods

you will call these methods inside getStatistic()